loader/loader.go

```diff
@@ -364,7 +364,10 @@ func (w *Worker) dispatchSQL(ctx context.Context, file string, offset int64, tab
 			}
 			lastOffset = cur
 
-			w.jobQueue <- j
+			select {
```

Comment:
What about load/loader.go:250? Should we `select ctx.Done` as well?

Reply:
yes, we should add it too, I'll fix it.